packages/app/src/cli/models/extensions/specifications/app_config_webhook.ts

```diff
@@ -73,11 +27,11 @@ export default appWebhooksSpec
 function simplifyWebhooks(remoteConfig: SpecsAppConfiguration) {
   if (!remoteConfig.webhooks?.subscriptions) return remoteConfig
 
-  remoteConfig.webhooks.subscriptions = mergeWebhooks(remoteConfig.webhooks.subscriptions)
+  remoteConfig.webhooks.subscriptions = mergePrivacyAndNonPrivacyWebhooks(remoteConfig.webhooks.subscriptions)
   return remoteConfig
 }
 
-function mergeWebhooks(subscriptions: WebhookSubscription[]): WebhookSubscription[] {
+function mergePrivacyAndNonPrivacyWebhooks(subscriptions: WebhookSubscription[]): WebhookSubscription[] {
```

Comment:
nit: I'm not a huge fan of introducing the term `NonPrivacy` to our codebase. It's not something that we're currently using (that I'm aware of) and someone looking at this code would wonder what non-privacy webhooks are. it's not a huge improvement but wdyt about `mergeAllWebhooks`?